src/main/java/com/amihaiemil/camel/StrictYamlSequence.java

```diff
@@ -0,0 +1,92 @@
+package com.amihaiemil.camel;
```

Comment:
@salikjan Add the copyright notice here, the same as it's done in other files :D